README.md

```diff
@@ -63,6 +63,14 @@ git checkout <release tag>
 
 Build and install using Meson and Ninja.
 
+The minimum version of Meson required to build HSE can be found in the root
+[meson.build](./meson.build). In there, you will find a `meson_version`
+keyword argument to the `project()` function at the beginnging of the file.
+
+If your system doesn't supply a Meson version new enough to build HSE, refer to
+the Meson
+[installation instructions](https://mesonbuild.com/Getting-meson.html).
+
```

Comment:
Can your meson scripts check the version and print an error message if the current version is too low?  That way people who missed this note won't spend an hour trying to decipher broken error messages...

Reply:
I'm pretty sure meson does that by default -- it's happened to me a fair number of times.

Reply:
Meson does that by default yes.

Reply:
You just have to have a meson version new enough to respect `meson_version`.

Reply:
Well, it blew up on me just yesterday with default ubuntu 18.04 package.  So again, can you put a check into the meson config to have it check itself and save our users from a load of unnecessary pain?

Reply:
This is message I get w/ an out of date meson.  Pretty clear to me Greg.  What are you seeing?
```
[0 chambers hse]$ meson setup build
The Meson build system
Version: 0.59.4
Source dir: /home/atomlinson/w/src/hse/hse
Build dir: /home/atomlinson/w/src/hse/hse/build
Build type: native build

meson.build:1:0: ERROR: Meson version is 0.59.4 but project requires >=0.60.0

A full log can be found at /home/atomlinson/w/src/hse/hse/build/meson-logs/meson-log.txt
```

Reply:
When Greg had trouble on Wednesday he was using Meson 0.45 from the Ubuntu repo. I am not sure when meson_version got added, but 0.45 might be too old, so Meson will happily try to parse a Meson file it doesn't understand which can lead to whacky errors which Greg saw.

Reply:
$ meson setup --buildtype=release -Dc_args='-march=native' -Dwerror=true -Dycsb=true /var/tmp/gbecker/hse
The Meson build system
Version: 0.45.1
Source dir: /home/gbecker/hse
Build dir: /var/tmp/gbecker/hse
Build type: native build
meson.build:258:28: ERROR: Expecting rparen got id.
or 'HdrHistogram_c' in get_option('force_fallback_for'))
^
For a block that started at 252,42
if not get_option('tools').disabled() and (not HdrHistogram_c_dep.found()
^
A full log can be found at /var/tmp/gbecker/hse/meson-logs/meson-log.txt
(1)

Not sure where I picked up version .45, when I install via apt now I get version .59.4.  Not clear why I got .45 the first time I ran apt install.  At any rate, if it happens to me it'll happen to someone else...